go/common/genesis/genesis.go

```diff
@@ -38,7 +51,7 @@ func writeCodes(codes map[common.Hash][]byte, out io.Writer) error {
 	return nil
 }
 
-func readCode(in io.Reader) ([]byte, error) {
+func ReadCode(in io.Reader) ([]byte, error) {
```

Comment:
This function used to be a utility function for a very specific purpose. By making it public and putting it in a common package, much more documentation would be required to describe what this function is supposed to do.